stable/shell/values.yaml

```diff
@@ -58,6 +58,15 @@ daemon:
     tag: "1.0.0"
     pullPolicy: "IfNotPresent"
 
+sshServer:
+  enabled: false
+
+  # can specifically set the port for the service NodePort (optional)
+  # port:
+
+  # base64 encoded public key
+  publicKey: ""
```

Comment:
`authorized_keys` file can contain multiple keys.
So either name that `authorizedKeys` and have it accept a base64 encoding of the entire file, or name it `publicKeys` which will be a list of base64 encoded strings , each of 1 publicKey